src/aria2-card.ts

```diff
@@ -176,13 +176,9 @@ export class Aria2Card extends LitElement {
     this.config = config
   }
 
-  override connectedCallback() {
-    super.connectedCallback();
-
-    if (this.hass) {
-      this.hass!.connection.subscribeEvents((e: any) => this._refresh(e.data.list), 'download_list_updated')
-      this.hass!.callService('aria2', 'refresh_downloads')
-    }
+  override firstUpdated() {
```

Comment:
Shouldn't the `super.firstUpdated();` be called?

Reply:
On the documentation the super is not called, so I think that is not mandatory.